doc/getting_started.ipynb

```diff
@@ -37,9 +37,9 @@
     "\n",
     "Launch jupyter notebook\n",
     "```\n",
-    "docker run -p 80:8888 -v pyannote-video/scripts:/scripts \\\n",
-    "                      -v pyannote-data:/data \\\n",
-    "                      -v pyannote-video/doc:/notebooks \\\n",
+    "sudo docker run -p 80:8888 -v $(pwd)/pyannote-video/scripts:/scripts \\ \n",
```

Comment:
The need for `sudo` really depends on your setup. 
For instance, in both my setups (Linux and OS X), I don't need to sudo.